README.md

```diff
@@ -15,8 +17,44 @@ By default, the following themes are used:
 
 ## Building
 ````
-
 ./gradlew build
 ./gradlew :auto-dark-mode:buildPlugin
-
 ````
+
+### Architecture support
+| Operating System | x86 Support        | x86_64 Support     |
+|------------------|--------------------|--------------------|
+| Windows          | :heavy_check_mark: | :heavy_check_mark: |
+| Mac OS           | :x:                | :heavy_check_mark: |
+| Linux            | :x:                | :heavy_check_mark: |
+
+### OS-dependent build components
+When Gradle builds the plugin, it will only be able to compile 
+native components for the operating system running the build.
+For example, Mac OS toolchains won't be available to someone 
+who is compiling on Windows. For this reason, this plugin depends on 
+artifacts built by a [custom GitHub Actions workflow](.github/workflows/libs.yml) for the platforms which cannot be compiled 
+in the given environment. [A custom Gradle plugin](buildSrc/src/main/groovy/UsePrebuiltBinariesWhenUnbuildablePlugin.groovy) 
```

Comment:
Thank you very much for providing this build documentation. Even though the plugin will issue a warning for it, it might be worth adding the information that a github access token is required for the artefacts to be downloaded from github.
Something along the line of
>For this you have to specify a valid [GithubAccessToken](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) using the `githubAccessToken` property in your local `gradle.properties` file. The access token only needs to have the permission to read repositories.

Reply:
Not a problem. I'm always happy to help document various components since it helps both us and any potential future contributors to find their way around.

Reply:
Completed in 078ed67d1530c874d515c458368af0927b058bd6.